src/main/java/io/debezium/connector/cassandra/CassandraConnectorConfig.java

```diff
@@ -288,6 +291,17 @@
 
     protected static final int DEFAULT_SNAPSHOT_FETCH_SIZE = 0;
 
+    private static final ConfigDefinition CONFIG_DEFINITION = CommonConnectorConfig.CONFIG_DEFINITION.edit()
+            .name("cassandra")
+            .type(
+                    COMMIT_LOG_RELOCATION_DIR,
+                    OFFSET_BACKING_STORE_DIR,
+                    SCHEMA_POLL_INTERVAL_MS,
+                    SNAPSHOT_POLL_INTERVAL_MS)
+            .create();
+
+    public static Field.Set ALL_FIELDS = Field.setOf(CONFIG_DEFINITION.all());
+
```

Comment:
Can we define it as something like `VALIDATION_FIELDS` instead of `ALL_FIELDS` here? It looks confusing since the fields added here does not include all fields in cassandra connector's config.

Instead of needing to create and use CONFIG_DEFINITION, we can try to do something like:

```
List<Field> validationFieldList = new ArrayList<> ();
validationFieldList.add(OFFSET_BACKING_STORE_DIR);
validationFieldList.add(COMMIT_LOG_RELOCATION_DIR);
...
public static Field.Set VALIDATION_FIELDS = Field.setOf(validationFieldList);
```